.changeset/rare-ducks-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/explorer": patch
+---
+
+World Explorer now includes filterable tables selector.
```

Comment:
```suggestion
Table selector of the Explore tab now has an input for searching/filtering tables by name.
```

Don't think we need to mention "World Explorer" here because the changeset is already "bound" to this package